cfpb/cfgov-refresh
# Set icon size and alignment to be geographically correct

Fixes misalignment in master

## Testing
 - Checkout and yarn run gulp build
 - Enter any zipcode on the find a housing counselor page
 - Zoom in and out, note that all markers should remained pointing to the same spot at every zoom level (and that spot should correspond to the address)
